src/pokete/classes/difficulty.py

```
@@ -133,6 +162,30 @@ def _update_director_state(self, last_rating):
         else:
             self.pacing_mode = PacingMode.NORMAL
 
+    def get_director_mood(self):
+        """Returns semantic mood based on internal state."""
+        if self.pacing_mode == PacingMode.RELIEF:
+            return "Merciful"
+        if self.pacing_mode == PacingMode.PRESSURE:
+            if self.win_streak > 3: return "Relentless"
```

Ruff (0.14.14)

[error] Multiple statements on one line (colon)

(E701)